lib/FrontendTool/FrontendTool.cpp

```diff
@@ -148,23 +148,21 @@ static bool emitMakeDependencies(DiagnosticEngine &diags,
       out << ' ' << escape(path);
     // Then print dependencies we've picked up during compilation.
     for (auto const &path :
-           reversePathSortedFilenames(depTracker.getDependencies()))
+         reversePathSortedFilenames(depTracker.getDependencies()))
```

Comment:
Nitpick: lost indentation. (Yes, Xcode is not helpful here.)

Reply:
Fixed. But git-clang-format indents it the other way.